### PR TITLE
Support for localization

### PR DESCRIPTION
**Pull request by user1m**
- added support for localization

### CODE REVIEW

**Comment by user1m** [comment]
@liliankasem @tombarton

**Comment by tombarton** [comment]
Hi @User1m, thank you so much for your contribution. It looks fantastic. My only request is to bring your changes inline with [Standard JS](https://standardjs.com/) formatting. I've updated the testing scripts to include standard, hence why the build is now failing.

Running `standard --fix` should resolve these.

File: index.js [hunk 1]
```diff
@@ -2,113 +2,113 @@ const builder = require('botbuilder')
...
-        if (!session.dialogData.reprompt) {
-          builder.Prompts.text(session, questions[index].question)
+        // Check if entities exists
+        if (session.dialogData.entities) {
+            // If the entities exists and it possesses the property for this question, send a confirm prompt
+            if (session.dialogData.entities.hasOwnProperty(questions[index].field)) {
+                var prompt = questions[session.dialogData.index].prompt
+                // Replace placeholder text with user data
+                prompt = prompt.replace('{' + questions[index].field + '}', session.dialogData.entities[questions[index].field])
+                builder.Prompts.confirm(session, session.localizer.gettext(session.preferredLocale(), prompt))
```
Comment (tombarton) [inline index.js]: I'm not 100% sure about this. I'm not too familiar with the localiser functions, but from what I've read it goes and fetches a string from a separate JSON file. The prompt variable here is being built from the question object that is fed into the module. If you wanted to go multilingual, wouldn't you fetch the different language version on line 27 before you add the entity data into the string?

I should be able to take a look into this in more detail on Monday.

**Review by tombarton — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (user1m) [reply index.js]: You're right. I wasn't sure what you were doing here at first but after looking closer I see you're replacing a placeholder variable with some user content. Pushed the change :)

**Review by user1m — COMMENTED** [review]
Using `standard --fix` and fixed entities issue

**Comment by user1m** [comment]
@tombarton changes made

**Comment by tombarton** [comment]
Perfect, thanks @User1m. I'll review tonight.

**Comment by user1m** [comment]
ping @tombarton

**Comment by tombarton** [comment]
Sorry @User1m, it's been a hectic couple of weeks. I'll make sure I review this in the next couple of days.

**Comment by tombarton** [comment]
Hi @User1m, I've been testing your contributions and they look good to me. I'm going to merge them into the `develop` branch and they will form part of the next release. Thank you for your contributions!

**Review by tombarton — APPROVED** [review]

**Pull request merged**